### PR TITLE
Postgres setup for local dev of ecr-viewer nextJS container

### LINKED ISSUE #974: Add local Postgres database for eCR Viewer
## Action Requested
Create a `docker-compose` script in the eCR viewer that:
- Starts the NextJS app
- Creates a Postgres database with a FHIR table (single table with `ecr_id` and `fhir` columns)

## Acceptance Criteria
The `docker-compose` script creates the app and database, and the database is seeded with some FHIR bundles for testing.

## Testing
Lots of local testing, and we may want to re-use the docker compose script for integration testing as well.

## Background & Context
The eCR Viewer will need a database option for local development. This may not be what we use long-term (that will depend on our pilot partner) but we need something to develop against in the interim.

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
This takes the ecr-viewer nextjs app and adds a local postgres DB for development purposes.

## Related Issue
Fixes #974 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/src/app/view-data/EcrSummary.tsx [hunk 1]
```diff
@@ -11,6 +9,9 @@ interface EcrViewerProps {
 const EcrSummary = (
     {fhirPathMappings, fhirBundle}: EcrViewerProps
 ) => {
+    console.log("BOOYAH:", fhirBundle);
+    console.log("GIVENNAME: ", fhirPathMappings.patientGivenName);
+    console.log("EVAL: ", evaluate(fhirBundle, fhirPathMappings.patientGivenName))
```
Comment (BobanL) [inline containers/ecr-viewer/src/app/view-data/EcrSummary.tsx:14]: Can we remove these logs?

File: containers/ecr-viewer/views/index.mustache
Comment (BobanL) [inline containers/ecr-viewer/views/index.mustache:1]: Can we delete this file now?

**Review by BobanL — COMMENTED** [review]
Code looks good. I left a couple of comments for clean up, but when I try to run locally I see this error when going to the /view-data page. It looks like it can't fetch that public fhirPath.yml file.
image

File: containers/ecr-viewer/src/app/view-data/EcrSummary.tsx (hunk 1 above)
Comment (nickbristow) [reply containers/ecr-viewer/src/app/view-data/EcrSummary.tsx]: you don't like them??

**Comment by nickbristow** [comment]
> Code looks good. I left a couple of comments for clean up, but when I try to run locally I see this error when going to the /view-data page. It looks like it can't fetch that public fhirPath.yml file. image

we get that error, then it goes away...not sure why

**Review by BobanL — APPROVED** [review]
🎉

**Pull request merged**